config/nav.yml

```diff
@@ -67,6 +67,7 @@ nav:
             - Creating a Service: developer/serving/services/creating-services.md
             - Service metrics: developer/serving/services/service-metrics.md
             - Configuring private Services: developer/serving/services/private-services.md
+            - Configuring Services custom Ingress Class: developer/serving/services/ingress-class.md
```

Comment:
not sure about how we want to capitalize ingress class - if it's not an API object the same way that Service etc is, I'd just keep it lower case?

Reply:
I am happy with lower case